tests/main/views/test_buyer_domains.py

```diff
@@ -158,6 +158,13 @@ def test_create_buyer_email_domain_fails_if_higher_level_domain_already_approved
 
 
 class TestListBuyerEmailDomains(BaseApplicationTest):
+    def setup(self):
+        super(TestListBuyerEmailDomains, self).setup()
+        with self.app.app_context():
+            # Remove any default fixtures so we can test pagination
```

Comment:
comment still specifically references pagination - more general would be good please as it's applicable for all tests.

Reply:
Fixed!